environment: "basic_environment"
rl_algorithm: "single_agent_qlearning"


max_num_episodes: 10

vis_update_params:
  every_n_episodes: 1 # Print episode info every_n_episodes
  every_n_steps: 1 # Print iteration info every_n_steps inside single episode
  print_reward_every_episode: True


test_mode_on: False # If test mode is on, the q_table is loaded and not saved nor updated, exploit is used always
load_q_table: False

# Q-learning Parameters:
q_learning_params:
  exp_exp_tradeoff: 0.5 # Random number ####
  learning_rate: 0.7  # Learning rate
  gamma: 0.5  # Discounting rate
# Exploration parameters
  epsilon: 1.0  # Exploration rate
  max_epsilon: 1.0  # Exploration probability at start
  min_epsilon: 0.01  # Minimum exploration probability
  decay_rate: 0.0001  # Exponential decay rate for exploration prob
